### ISSUE TITLE
Show the project and options used when a punchcard is generated

### ISSUE THREAD

**Issue by icyflame** [guanqun/git-punchcard-plot#10]
I just went back to the folder and saw a lot of images, and I have no clue what repositories they correspond to. We can put some text on the image I think, the repository's local path, values of the provided options (if the user wants it). What do you think?

**Comment by guanqun** [comment]
Yeah, definitely, we can also have an option, probably "title" to overwrite the default if a user wants it.
